File: 2D_amber/Assets/Scripts/JS/World/WorldTitle.js
#pragma strict

class WorldTitle extends World{
  var btns : GameObject ;
  var anyKeyText : GameObject ;
  
  function Update(){
    // 按下任意鍵
    if(Input.anyKey){
      btns.SetActive(true) ;
      anyKeyText.SetActive(false) ;
    }
  }
}

File: 2D_amber/Assets/Scripts/CSharp/World/WorldTitle.cs
using UnityEngine;
using System.Collections;

public class WorldTitle : World {
	public GameObject btns ;
	public GameObject anyKeyText ;
  
	void Update(){
		// 按下任意鍵
		if(Input.anyKey){
			btns.SetActive(true) ;
			anyKeyText.SetActive(false) ;
		}
	}
}
